recipe/meta.yaml

```
@@ -73,10 +73,12 @@ test:
     - jinja2
     - cffi
     - scipy
+    # At least NumPy 1.26.0 has an issue with arctanh on glibc<=2.17.
+    # ref: https://github.com/conda-forge/numba-feedstock/pull/128#issuecomment-1795031396
```

Let's link the NumPy issue too for clarity

```suggestion
    # ref: https://github.com/conda-forge/numba-feedstock/pull/128#issuecomment-1795031396
    # ref: https://github.com/numpy/numpy/issues/25087
```

---

Let's go for just the upstream issue link (that one has references to everything else).